File: src/main/java/level_a/java/GravityFlip.java
package level_a.java;

import java.io.*;
import java.util.Arrays;

public class GravityFlip {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int cols = Integer.parseInt(br.readLine());
        String[] col = br.readLine().split("\\s+");
        int[] heights = new int[cols];
        for (int i = 0; i < cols; i++)
            heights[i] = Integer.parseInt(col[i]);
        Arrays.sort(heights);
        for (int i = 0; i < heights.length; i++)
            System.out.print(heights[i] + " ");
    }
}

File: src/main/java/level_a/kotlin/GravityFlip.kt
package level_a.kotlin

import java.io.BufferedReader
import java.io.InputStreamReader
import java.lang.StringBuilder

fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    br.readLine()
    val row = br.readLine().split("\\s+".toRegex())
    val sortedRow = row.sortedBy { i -> i.toInt() }
    val sb = StringBuilder()
    for (i in sortedRow)
        sb.append("$i ")
    sb.deleteCharAt(sb.length - 1)
    print(sb.toString())
}